packages/moonstone/FormCheckbox/FormCheckbox.js

```diff
@@ -1,9 +1,12 @@
 /**
- * Provides Moonstone-themed checkmark icon inside a circle, primarily used inside the
+ * Moonstone styled checkmark icon inside a circle, primarily used inside the
```

Comment:
Noticed the PropTypes for this on aren't documented any reason for that?

Reply:
Added the description for `children`, but the `css` prop is private so we are leaving it as is.